dags/hello_world/hello_world.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.utils.dates import days_ago
+from airflow.models.taskinstance import TaskInstance
+from returns.curry import curry
+
+
+@curry
```

Comment:
To pass return values from one task in a graph to the next, they have to be pushed to and pulled from [xcom](https://airflow.apache.org/docs/apache-airflow/stable/concepts/xcoms.html) (given, that they are not too big)